### PR TITLE
Automatically select first contact of group

### PR DESCRIPTION
**Pull request by Henni**
If the URL doesn't contain a contact id, the first contact of the contactList is chosen.

### CODE REVIEW

File: js/public/script.js [hunk 1]
```diff
@@ -254,6 +254,8 @@ app.controller('contactlistCtrl', ['$scope', 'ContactService', '$routeParams', f
 		addContact : t('contactsrework', 'Add contact')
 	};
 
+	ctrl.contactList = [];
```
Comment (DeepDiver1975) [inline js/public/script.js]: Where is contactList getting assigned?

**Comment by DeepDiver1975** [comment]
This is required to get #59 fixed

File: templates/contactList.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <button class="app-content-list-button" type="button" name="button" ng-click="ctrl.createContact()">{{ctrl.t.addContact}}</button>
 <div class="app-content-list-item"
-	 ng-repeat="contact in ctrl.contacts | contactGroupFilter:ctrl.routeParams.gid | orderBy:'fullName()' track by contact.uid()"
+	 ng-repeat="contact in ctrl.contactList = (ctrl.contacts | contactGroupFilter:ctrl.routeParams.gid | orderBy:'fullName()') track by contact.uid()"
```
Comment (DeepDiver1975) [inline templates/contactList.html]: aha

**Comment by DeepDiver1975** [comment]
works :+1:

**Comment by DeepDiver1975** [comment]
hmmm . when selecting another group it requires 2 clicks to select the first contacts

**Comment by Henni** [comment]
I'll fix this

**Comment by DeepDiver1975** [comment]
> I'll fix this

I'm already at it ....

**Comment by Henni** [comment]
:+1:

**Pull request merged**